modules/virt-hot-plugging-a-virtual-disk.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-plugging-a-virtual-disk_{context}"]
+= Hot-plugging a virtual disk
+
+Hot-plug virtual disks that you want to add to or remove from a virtual machine instance while a virtual machine is running.
+
+.Prerequisites
+* You must have a running virtual machine to hot-plug a virtual disk.
+* You must have at least one data volume or persistent volume claims (PVC) available for hot-plugging.
+
+To hot-plug a virtual disk using the CLI, use the `addvolume` command with `virtctl`.
+
+.Procedure
+
+. Run the following command:
+
+----
+$ virtctl addvolume <virtual-machine|virtual-machine-instance> --volume-name=<datavolume-or-PVC-to-hot-plug> [--persist] [--serial=<label-name>]
+----
+
+* The `addvolume` command creates an attachment pod to attach the hot-plugged disk.
```

Comment:
This is an implementation detail, the addvolume command doesn't actually do that, the addvolume command triggers a disk hotplug, and as part of that process an attachment pod is created.

Reply:
Noted and deleted. Wasn't sure about including this and won't if it's not technically correct and will open up more questions on the user's end.